### DISCUSSION TITLE
Filter is Not working with enums/SmartEnums

### DISCUSSION THREAD

**Discussion by CodeWrapper12** [Biarity/Sieve#204] [Q&A]
Filter is Not working with enums/SmartEnums

**Comment by CodeWrapper12** [comment]
is any otherways to apply sieve filtering on smartenum

**Reply by jansonjc** [reply]
I'm also facing the same issue. I'm using Ardalis SmartEnum as a property on the entity. It uses string type for value (to store the code) and that is what get persisted in the database column.
